src/alchemtest/gmx/__init__.py

```diff
@@ -3,3 +3,5 @@
 """
 
 from .access import load_benzene
+from .access import load_expanded_ensemble_case_1
```

Comment:
let's do imports on a single statement (not sure why we had it differently elsewhere... maybe @dotsdl 's preference??)
```python
from .access import (load_benzene, load_expanded_ensemble_case_1,
                     load_expanded_ensemble_case_2, )
```